### ISSUE TITLE
Документ Установка цен(создание)

### ISSUE THREAD

**Issue by Linda4ka** [Contabilizare/c5#44]
Можно ли в табличную часть данного документа внести окошко с Оптовой ценой и окошко с Ценой реализации(Продажной ценой)? Это будет необходимо для предприятий, которые имеют и магазин и оптовые базы одновременно, т.е. производят оптовую и розничную продажи.Спасибо!

**Comment by SvetlanaRes** [comment]
У меня был такой же вопрос. Спасибо, документация помогла. В табличной части отображаются только типы цен с признаком "Базовый".

**Comment by Nullarity** [comment]
> У меня был такой же вопрос. Спасибо, документация помогла. В табличной части отображаются только типы цен с признаком "Базовый".

Не, не только, так тоже можно:
![image](https://user-images.githubusercontent.com/28710693/118997825-f1346180-b956-11eb-8984-d94edf944d8e.png)

**Comment by SvetlanaRes** [comment]
Да, получилось.  Но только в этих 2-х вариантах: Базовый и Процент на товар. Остальные не отображаются.
